Sources/ArgumentParser/Parsing/ParserError.swift

```diff
@@ -25,7 +25,7 @@ enum ParserError: Error {
   case duplicateExclusiveValues(previous: InputOrigin, duplicate: InputOrigin, originalInput: [String])
   /// We need a value for the given key, but it’s not there. Some non-optional option or argument is missing.
   case noValue(forKey: InputKey)
-  case unableToParseValue(InputOrigin, Name?, String, forKey: InputKey)
+  case unableToParseValue(InputOrigin, Name?, String, forKey: InputKey, customMessage: String?)
```

Comment:
`customMessage` could have a default `nil` value so that you don't need to specify it every time

Reply:
Adding a default value to the `customMessage` parameter also reduced the file footprint of this PR.